src/Altinn.Platform/Altinn.Platform.Events/Events/Altinn.Platform.Events.csproj

```diff
@@ -44,14 +44,6 @@
     <CodeAnalysisRuleSet>..\..\..\..\Altinn3.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="Migration\_erase\" />
```

Comment:
Hva er grunnen til at dette slettes?

Reply:
Just tidying up in the project file. The entries were unnecessary.